include/seqan3/search/algorithm/detail/search_trivial.hpp

```diff
@@ -157,7 +166,15 @@ template <bool abort_on_hit, typename index_t, typename query_t, typename delega
 inline void search_trivial(index_t const & index, query_t & query, search_param const error_left,
                            delegate_t && delegate) noexcept(noexcept(delegate))
 {
-    search_trivial<abort_on_hit>(index.begin(), query, 0, error_left, delegate);
+    search_trivial<abort_on_hit>(index.begin(), query, 0, error_left, false, false, delegate);
+}
+
+//quickfix for unit test function calls
```

Comment:
Is this function just a temporary workaround or is it supposed to be merged?

Reply:
I did not want to touch the unit test in this PR. I will do another PR afterwards were I modify the unit test (and also add a test for Insertion after a Deletion substituting a mismatch).

Reply:
Then I'd recommend moving the new function also to the new PR because it is not needed here (and then add doxygen there).

Reply:
I just noticed it is only one function call i will change that and add the unit test in this PR too.